src/Serilog.Sinks.RollingFile/project.json

```diff
@@ -0,0 +1,29 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The rolling file sink for Serilog - Simple .NET logging with fully-structured events",
+  "authors": [ "Serilog Contributors" ],
+  "tags": [ "serilog", "file", "io", "rolling" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png",
+  "dependencies": {
+    "Serilog": { "target": "project" },
+    "Serilog.Sinks.IOFile": { "target": "project" }
+  },
+  "compilationOptions": {
+    "keyFile": "../../assets/Serilog.snk"
+  },
+  "frameworks": {
+    "net45": {
+    }, 
+    "dotnet5.4": {
```

Comment:
I think this should be multi-targeted to `dotnet5.1` as well. `System.IO.FileSystem.Primitives` has both targets.